Error: HTTPSConnectionPool(host='api.tibber.com', port=443): Max retries exceeded with url: /v1-beta/gql (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7fff3cbdd3d0>: Failed to establish a new connection: [Errno 113] No route to host'))
